index.html

```diff
@@ -1,23 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="utf-8">
-    <meta name="description" content="Your description goes here">
-    <meta name="keywords" content="one, two, three">
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="Your description goes here">
+  <meta name="keywords" content="one, two, three">
 
-    <title>Blank Template</title>
+  <title>Blank Template</title>
 
-    <!-- external CSS link -->
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+  <!-- external CSS link -->
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 
-  </head>
+</head>
 
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+<body>
+  <div id="wholepage">
+
+    <!-- First Section -->
+    <div class="firstbox">
+      <p class="firstp">Relaxr</p>
+      <div class="wrapper">
+        <p class="secondp">Get a piece of mind with a single tap</p>
+        <a class="firstbutton" href="#">Get it Now</a>
+      </div>
+    </div>
+
+    <!-- Second Container -->
+
+    <div class="secondbox">
+      <h1 class="firstheader">Benefits</h1>
+      <p class="assistant">The perfect personal assistant. Relaxr does your job</br>for you so you can enjoy life as it is meant to be.</p>
+      <ul class="list">
+        <li>Schedule meetings for you</li>
+        <li>Excel automation to complete your</br>work for you</li>
+        <li>Responds to emails on your behalf</li>
+        <li>Does all your work for you with our </br>revolutionary Al technology</li>
+      </ul>
+    </div>
+
+    <!-- Third Section -->
+
+    <div class="thirdbox">
+      <p class="water">"Relaxr changed my life. I've been able</br> to travel the world, spend limited time </br>working and my boss keeps
+        thanking</br>
+        me for crushing work."</p>
+      <h1 class="name">- Amanda, Intuit</h1>
+    </div>
+
+    <!-- Fourth Section -->
+    <div class="fourthbox">
+      <a class="secondbutton" href="#">Sign Up Now!</a>
     </div>
-  </body>
+
+    <!-- Last Section -->
+    <div class="socialmediabox">
+      <h1 class="last">Relaxr</h1>
+      <img src="https://raw.githubusercontent.com/ericmasiello/FEWD-Assignment2/master/images/facebook.png" height="20px" weight="20px"
```

Comment:
Avoid styling elements (height and width) inline like this. Instead, use CSS to target them and set the width and height there.